tic_tac_toe/tic_tac_toe.py

```diff
@@ -0,0 +1,13 @@
+class TicTacToe:
+    def __init__(self):
+        self.board = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+
+    def display_board(self, board):
```

Comment:
This might be a Python thing you're not used to—it threw me off at first, coming from Ruby—but in case you're not aware, you aren't actually using the `board` argument that's getting passed to this function. The body of the function is referencing the board stored on this class instance (`self.board`), which is not necessarily the same as the `board` argument you're passing in. (In your tests, you happen to be passing that same object, i.e. `ttt.display_board(ttt.board)`, but you're still dropping that argument on the floor once you get into this method.)

In case this doesn't make sense, try this out:
```python
ttt = TicTacToe()
ttt.display_board(["X", "X", "X", "X", "X", "X", "X", "X", "X"]
```

Reply:
Whoa. That's crazy. I totally see it now, but yeah, that was my Ruby instinct kicking in.

That's easily fixed by just referencing the argument in the function body rather than the the class instance, but that feels weird. Trying to pass in and modify `self.board` throws errors. It works if I set the board as a class attribute rather than a class instance, but that also feels weird. I think I need to spend some time reading up on Object Orientation in Python.